app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,24 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var MAILCHECK_ON = 'true';
+      var abChoose;
+      var abData = {
+        isMetricsEnabled: !! this.metrics._able,
+        uuid: this.metrics._uuid
+      };
+
+      var mailcheckParamChoice = Url.searchParam('mailcheck', this.window.location.search);
+      // the window parameter will override any ab testing features
```

Comment:
@dannycoates what do you prefer / advise, should the FORCED choice via the window query (`&mailcheck=true`) be in the experiment code or outside of it?